lib-ee/emqx_ee_connector/src/emqx_ee_connector_influxdb.erl

```diff
@@ -414,7 +421,7 @@ parse_batch_data(InstId, BatchData, SyntaxLines) ->
         fields := [{binary(), binary()}],
         measurement := binary(),
         tags := [{binary(), binary()}],
-        timestamp := binary()
+        timestamp := list() | integer()
```

Comment:
Nit: if this `list()` means the processed template, maybe a more precise type would be:

```suggestion
        timestamp := emqx_plugin_libs_rule:tmpl_token() | integer()
```